### ISSUE TITLE
L2T17_Store.html alt attributes update

### ISSUE THREAD

**Issue by BrettlesP96** [BrettlesP96/Hyperion-Projects#1]
I would want to add some alt attributes to my image tags to improve the website's accessibility for users with screen readers or in cases where images cannot be loaded.
